pycryptobot.py

```diff
@@ -593,10 +620,18 @@ def executeJob(sc=None, app: PyCryptoBot=None, state: AppState=None, trading_dat
 
                     if not app.isVerbose():
                         Logger.info(formatted_current_df_index + ' | ' + app.getMarket() + ' | ' + app.printGranularity() + ' | ' + price_text + ' | BUY')
+                        
+                        trading_dataCopy = trading_data[trading_data['date'] <= current_sim_date].copy()
+                        ta = TechnicalAnalysis(trading_dataCopy)
 
-                        bands = technical_analysis.getFibonacciRetracementLevels(float(price))
+                        bands = ta.getFibonacciRetracementLevels(float(price))
```

Comment:
aren't we missing `technical_analysis.addAll()` call around here, like in other places where TA is added to the data frame?

Reply:
the technical_analysis is carried out when smartswitching occurs, it should technically only happen once for each dataframe (1h & 15m), the way the getFibonacciRetracementLevels works is it takes the technical_analysis finds the last row and returns that, to make it work correctly for sims I have to pass in a subset of the data using the current sim date this then means the last rows corresponds to the date being used, the technical_analysis.addAll() has already been run further up the code so no need to run it again here.

Reply:
thanks, no problem, then =)